tests/test_engine_initialization.py

```
@@ -244,25 +268,33 @@ async def _fake_get_response(messages, stream=None, stream_callback=None, **kwar
-        category=ErrorCategory.BAD_REQUEST,
-        retryable=False,
+    retryable_error = LLMError(
+        message="stream disconnected after partial output",
+        category=ErrorCategory.NETWORK,
+        retryable=True,
         provider="openai",
         model="gpt-test",
     )
+    current_error: LLMError | None = None
 
     async def _fake_get_response(messages, stream=None, stream_callback=None, **kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_fake_get_response`

Add return type annotation: `str`

(ANN202)